extensions/binding/org.eclipse.smarthome.binding.astro/README.md

```diff
@@ -10,17 +10,19 @@ This binding supports two Things: Sun and Moon
 
 ## Discovery
 
-Discovery is not necessary, because all calculations are done within the binding.
+You can use the auto discovery if you have a location set in "Configuration - System - Regional Settings - Location". This will automatically create a "Local Sun" and a "Local Moon" item for this location.
```

Comment:
Simply say "if you have a system location set". There might be many ways how to do that, it is not up to the binding to know how.